modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinCleanupTest.java

```diff
@@ -55,6 +56,31 @@
     private Stage stage;
     private Pane root;
 
+// ------------------ ListCell
+
+    @Test
+    public void testListCellReplaceListViewWithNull() {
+        ListCell<Object> cell =  new ListCell<>();
+        ListView<Object> listView = new ListView<>();
+        cell.updateListView(listView);
+        installDefaultSkin(cell);
+        cell.updateListView(null);
+        listView.setFixedCellSize(100);
```

Comment:
I presume this test is just checking that no exception is thrown? If so, can you add a comment to that effect?

Reply:
yeah, exactly :) 

Was NPE before fix (due to skin listening to property of previous listView and trying to access the current which is null). Added comment as requested